src/hooks/useTransactions/useTransactions.ts

```diff
@@ -51,7 +54,7 @@ const formatTransactions = (txs: Transaction[], walletAddress: string): FormatTr
       date,
       dateFromNow: dayjs(date).fromNow(),
       fee: fromBaseUnit(tx.fee, 18),
-      chain: 'ETH' /* TODO: get chian from asset service */
+      chain: tx.chain
```

Comment:
This might not behave exactly as we want with the generics. Using `ChainTypes` the chain will be the string union instead of the actual specific chain name correct?

Reply:
nah in this case `tx` is data rather than a type so it's one of those chains in the `ChainTypes` enum